packages/ui/src/components/YouVersionAuthButton.tsx

```diff
@@ -72,7 +83,7 @@ export interface SignInButtonProps
  *   return (
  *     <div>
  *       {auth.error && <p className="text-red-600">Error: {auth.error.message}</p>}
- *       <SignInButton
+ *       <YouVersionAuthButton
  *         requiredPermissions={[SignInWithYouVersionPermission.bibles]}
  *         optionalPermissions={[SignInWithYouVersionPermission.highlights]}
```

Comment:
I believe `requiredPermissions` and `optionalPermissions` have been collapsed into a single `permissions` prop.